WeScan/Scan/RectangleFeaturesFunnel.swift

```diff
@@ -54,9 +59,15 @@ final class RectangleFeaturesFunnel {
     /// The value in pixels used to determine if two rectangle match or not. A higher value will prevent displayed rectangles to be refreshed. On the opposite, a smaller value will make new rectangles be displayed constantly.
     let matchingThreshold: CGFloat = 40.0
     
+    /// The number of similar rectangles that need to be found to auto scan.
+    let autoScanThreshold = 35
+    
     /// The minumum number of matching rectangles (within the `rectangle` queue), to be confident enough to display a rectangle.
-    let minNumberOfMatches = 2
-
+    let minNumberOfMatches = 3
+    
+    /// The number of times the rectangle has passed the threshold to be auto-scanned
+    var thresholdPassed = 0
```

Comment:
The name of this variable isn't very explicit. Maybe `currentNumberOfAutoScanPass` would be better?